README.md

```diff
@@ -291,3 +291,30 @@ const unleash = new UnleashClient({
 **NOTES: ⚠️**
 If `bootstrapOverride` is `true` (by default), any local cached data will be overridden with the bootstrap specified.   
 If `bootstrapOverride` is `false` any local cached data will not be overridden unless the local cache is empty.
+
+
+## Internal state
+
+Use of event-based communication with the client is recommended, but it's not always possible. One example is a reuse of existing and initialized client.
+To do this, you can get the state with `client.getState()` method. The `getState().status` is changing according to the followind diagram:
+
+```mermaid
+  graph LR;
+      created-->initialized;
+      created-->|READY|bootstrapped;
+      initialized-->starting;
+      bootstrapped-->healthy;
+      starting-->|READY|healthy;
+      healthy-->stopped;
+      stopped-->starting;
+```
+
+- `created` - The initial state of the client
+- `initialized` - The client has read configuration and local cached data
+- `bootstrapped` - SDK is configured to work offline and has been bootstrapped with feature toggles
+- `starting` - flags where not updated from Unleash yet
+- `healthy` - all flags are up to date
+- `stopped` - the client has been stopped manually and will not check for updates
+- `error` - an error has occurred while loading flags
```

Comment:
I find this a bit confusing and I believe we should align on these states as all our proxy-sdks should behave more or less the same way, so I believe this change is an important decision.